subprojects/docs/src/docs/userguide/migrating_from_maven.adoc

```diff
@@ -0,0 +1,518 @@
+Despite this, migrations can be surprisingly easy because Gradle follows many of the same conventions as Maven and dependency management works in a similar way.
+
+One especially useful feature for understanding your new Gradle build is link:https://scans.gradle.com/[build scans]: these are  web-based snapshots of a given build that allows collaborative debugging and fine-grained performance analysis.
+For example, here's the link:https://scans.gradle.com/s/u7uyv3vuyhrig/timeline[build timeline for the Groovy build]:
+
+image:groovy-build-scan.png[]
+
+You can use build scans to debug dependency resolution, Gradle task execution, and many other aspects of your build.
+
+Once you've decided to go ahead with the migration, what should you do first? The best starting point is _recording the inputs and outputs_ of your Maven build, so that you can verify that your new Gradle build is functionally equivalent.
+
+
+[[migmvn:automatic_conversion]]
+== Performing an automatic conversion
+
+Not only will the <<build_init_plugin#build_init_plugin,Gradle `init` task>> allow you to create a new skeleton project, but it will also automatically convert an existing Maven one to Gradle.
+All you have to do is run the command
+
+[listing.terminal]
+----
+$ gradle init
+----
+
+from the root project directory and let Gradle do its thing.
+That basically consists of parsing the existing POMs and generating the corresponding Gradle build scripts.
+Gradle will also create a settings script if you're migrating a <<multi_project_builds#multi_project_builds,multi-project build>>.
+
+You'll find that the new Gradle build includes the following:
+
+ * All the custom repositories that are specified in the POM
+ * Your external and inter-project dependencies
+ * The appropriate plugins to build the project (limited to one or more of the <<maven_plugin#maven_plugin,Maven>>, <<java_plugin#java_plugin,Java>> and <<war_plugin#war_plugin,War>> Plugins)
+ 
+See the <<build_init_plugin#sec:pom_maven_conversion_,Build Init Plugin chapter>> for a complete list of the automatic conversion features.
+
+One thing to bear in mind is that assemblies are not automatically converted.
+They aren't necessarily problematic to convert, but you will need to do some manual work.
+
+If you're lucky and don't have many plugins or much in the way of customisation in your Maven build, you can simply run
+
+[listing.terminal]
+----
+$ gradle build
+----
+
+once the migration has completed.
+This will run the tests and produce the required artifacts without any extra intervention on your part.
+
+[[migmvn:migrating_deps]]
+== Migrating dependencies
+
+Gradle's dependency management system is more flexible than Maven's, but it still supports the same concepts of repositories, declared dependencies, scopes (<<managing_dependency_configurations#managing_dependency_configurations,dependency configurations>> in Gradle), and transitive dependencies.
```

Comment:
We could optionally link "repositories", "declared dependencies" and "transitive dependencies" to relevant docs if you like.

Reply:
I don't like too many links as they tend to distract the reader. In this particular case, I don't see a lot of value since I'm assuming that, as Maven users, they're already familiar with the terms.